flake-modules/packages.nix

```diff
@@ -1,14 +1,17 @@
 {
   perSystem =
     {
-      pkgs,
+      pkgsUnfree,
       config,
       rawModules,
       helpers,
       ...
     }:
     {
-      packages = import ../docs { inherit rawModules pkgs helpers; };
+      packages = import ../docs {
+        inherit rawModules helpers;
+        pkgs = pkgsUnfree;
+      };
```

Comment:
This change seems unrelated. What's the motivation?

Reply:
we had in pkgsDoc the following: `        config.allowUnfree = true;`

Reply:
> we had in pkgsDoc the following: ` config.allowUnfree = true;`

Ah I missed that, thanks.

Do you know _why_ we need unfree? This is a good opportunity to add a comment explaining.

Reply:
Because some plugins/dependencies are unfree I believe, and building the docs evaluates them

Reply:
Maybe a comment along these lines then?
```nix
# Building the docs evaluates each plugin's default package, some of which are unfree
```